work_with_shared_resources/using-shared-resource-csi-driver.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="shared-resource-csi-driver"]
+= Shared Resource CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: shared-resource-csi-driver
+
+toc::[]
+
+
+[role="_abstract"]
+As a cluster administrator, you can use the Shared Resource Container Storage Interface (CSI) Driver in any Kubernetes object that uses the volume type as `csi`. {builds-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, pods and other Kubernetes types that expose volume mounts, and {ocp-product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: 
+
+* `SharedSecret` custom resource for `Secret` objects
+* `SharedConfigMap` custom resource for `ConfigMap` objects
+
+
+include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-sharing-secrets-across-namespaces.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-sharing-configmaps-across-namespaces.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-using-a-sharedconfigmap-object-in-a-pod.adoc[leveloffset=+1]
```

Comment:
Do we need a similar module for using a `SharedSecret` in a pod?

Reply:
If there is any information that needs to be covered for `SharedSecret` , we can add a section or a note depending on the content. 
If it's a procedure- let's add a section but if there's a small piece of information, a note should be fine I feel. WDYT?

Reply:
The content feels superficially similar, however the name of the objects are different - `SharedSecret` for `Secret` objects, `SharedConfigMap` for `ConfigMap` objects. This impacts other parts of the procedure - particularly the RBAC yaml that we ask admins to create (`ClusterRole`, `Role`, etc.).

Le'ts start with two sections and iterate on improvements from there.